ksp_plugin/plugin.cpp

```diff
@@ -386,6 +388,24 @@ void Plugin::InsertOrKeepVessel(GUID const& vessel_guid,
           << parent_index;
 }
 
+void Plugin::InsertUnloadedPart(
+    PartId part_id,
+    GUID const& vessel_guid,
+    RelativeDegreesOfFreedom<AliceSun> const& from_parent) {
+  not_null<Vessel*> vessel = find_vessel_by_guid_or_die(vessel_guid).get();
+  CHECK(is_new_unloaded(vessel));
+  RelativeDegreesOfFreedom<Barycentric> const relative =
+      PlanetariumRotation().Inverse()(from_parent);
+  ephemeris_->Prolong(current_time_);
+  AddPart(
+      vessel,
```

Comment:
The indent is weird, I'd prefer to move this to the previous line and split the last parameter.